### DISCUSSION TITLE
Lyric round auto mode

### DISCUSSION THREAD

**Discussion by Slug-Life** [d0n4v4nb3ck3r/homebridge-honeywell-home#773] [Q&A]
After looking through some of the closed issues it seems like auto mode for the Lyric round is supported. However I only have access to off, heat, and cool. Is this a bug or a limitation of the plugin? Thanks!

**Comment by donavanbecker** [comment]
The plugin looks at what your thermostat says is possible and then displays it. So based off how your thermostat is setup you probably can't display `Auto`

**Comment by Slug-Life** [comment]
I appreciate the response. To elaborate, I had auto mode working with native Homekit integration, and its available in the reside app. After looking at the logs if I go into the Resideo app and enable auto mode from there I noticed an error: 

This plugin generated a warning from the characteristic 'Target Heating Cooling State': characteristic value 3 is not contained in valid values array. See https://homebridge.io/w/JtMGR for more info.

This seems similar to issue #554

**Reply by donavanbecker** [reply]
Can you open an issue and get me some debug logs of what the API is sending back.
